mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -386,12 +387,14 @@ CLASS net/minecraft/class_1297 net/minecraft/entity/Entity
 		COMMENT axis-aligned integer box is fully loaded in the world.
 	METHOD method_35049 getRemovalReason ()Lnet/minecraft/class_1297$class_5529;
 	METHOD method_36209 onRemoved ()V
+	METHOD method_36361 isGlowingLocal ()Z
```

Comment:
isn't this getting the actual glowing field than allowing considered glowing by effect etc?

Reply:
isGlowing (existing function) checks getFlag for clients, and glowing field for servers, last time i checked. This one is the actual getter for the glowing field